pycqed/measurement/pulse_sequences/fluxing_sequences.py

```diff
@@ -548,8 +765,13 @@ def cphase_seqs(qbc_name, qbt_name, hard_sweep_dict, soft_sweep_dict,
             print(f'max_pulse_length = {max_flux_length*1e9:.2f} ns, '
                   f'from pulse dict.')
     # add buffers to this delay
-    delay = max_flux_length + flux_pulse.get('buffer_length_start', 0) + \
-        flux_pulse.get('buffer_length_end', 0)
+    nr_flux_buffer = 4 if flux_pulse['pulse_type']=='BufferedNZHalfwayPulse' \
+        else 2 #for NZ pulse we have four buffers
```

Comment:
add comment to make clear that this only applies to FLIP gates

Reply:
Done!